lib/main_common.pm

```diff
@@ -708,7 +708,7 @@ sub load_x11regression_other {
         loadtest "x11/gnomeapps/gnome_documents";
         loadtest "x11regressions/totem/totem_launch";
     }
-    if (check_var('DISTRI', 'sle') && sle_version_at_least('12-SP2')) {
+    if (is_sle && sle_version_at_least('12-SP2') && !sle_version_at_least('15')) {
```

Comment:
can you comment here why the tests must not be called for >=15?

Reply:
As mentioned in the commit message: `- shotwell cases were dropped in SLED15`
We have replaced shotwell with gnome-photos in SLE15, I will add the comment on Monday~

Reply:
Updated.